environment.yml

```diff
@@ -37,7 +38,7 @@ dependencies:
   - mysql
   - mysql-connector-c
   - nanoflann
-  - nlohmann_json
+  - nlohmann_json>=3.9.1
```

Comment:
Needed for the `ordered_json` functionality. I'm not sure that this is strictly needed (I initially added it because I was initially thinking that ordered mattered, since it does for actual PDS4 output, but at this point, I don't think it's needed for what this will actually be used for (populating a template)), but it did make it easier to make sure that the converted JSON matched the XML input.

Reply:
We shouldn't need ordering because the templating doesn't use it.